packages/shared/src/components/onboarding/MemberAlready.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ClickableText } from '../buttons/ClickableText';
+import { AuthTriggers } from '../../lib/auth';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+interface ClassName {
+  container?: string;
+  login?: string;
+}
+
+interface MemberAlreadyProps {
+  className?: ClassName;
+  onLogin?(): void;
+}
+
+export function MemberAlready({
+  className = {},
+  onLogin,
+}: MemberAlreadyProps): ReactElement {
+  const { showLogin } = useAuthContext();
+
+  const onClick = () => {
+    if (onLogin) return onLogin();
+
+    return showLogin(AuthTriggers.Onboarding, { isLogin: true });
+  };
```

Comment:
Is this supposed to return when `onLogin` is provided as prop?

If yes I would even go as far to make `onLogin` required and explicitly pass `onLogin={() => showLogin(AuthTriggers.Onboarding, { isLogin: true }))` on places where it is needed. 

Think it will be better then component having hidden logic that will get overriden if you provide a prop.

Reply:
Makes sense. Let's do that.